.github/workflows/ruby.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will download a prebuilt Ruby version, install dependencies and
+# run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
These `branches` means that the jobs run only for updates on the branch `master`. 
So these `branches` are not needed, I think.

Reply:
Ah, I misunderstood this settings. It works as we expected.